BankSDK/GiniBankSDKExample/GiniBankSDKExample/Utils/Localization/Strings/de.lproj/SelectAPI.strings

```diff
@@ -0,0 +1,13 @@
+/* 
+  SelectAPI.strings
+  GiniBankSDKExample
+
+  Created by Valentina Iancu on 13.09.23.
+  
+*/
+
+"WELCOME_TITLE" = "Willkommen bei Gini";
```

Comment:
[Suggestion]: SDK we use “import.data.error.title” = “Invalid document”;
here in the example app:
“WELCOME_TITLE” = “Welcome to Gini”;

:sweat_smile: I don’t mind - but I think we need to keep in sync the style of localized keys 🙈

Reply:
I've also updated to the new german translation for the photo payment button and the alternative text.